project.clj

```diff
@@ -1,7 +1,7 @@
 (def ks-version "1.2.0")
 (def tk-version "1.2.0")
 
-(defproject puppetlabs/trapperkeeper-metrics "0.1.3-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper-metrics "0.2.0-SNAPSHOT"
```

Comment:
I think there's a parameter on the Jenkins job that lets you specify what version should be bumped. I've also never done a Jenkins y-release, but I think using that parameter is the way to go